cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -381,6 +382,21 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 		}
 	}
 
+	groupVersion = "batch/v2alpha1"
```

Comment:
I didn't notice that earlier, but I see this is a pattern, we should think about changing. Using strings for groupVersion is not the recommended approach. But that's an overall problem with this file, can be addressed later on.